docs/guides/validation-user-guide.md

```diff
@@ -4,4 +4,497 @@ headline: Validation User Guide
 bodyclass: docs
 layout: docs
 ---
-<p class="coming-soon">Coming soon...</p>
+
+Spine uses Protobuf for defining domain models. It also helps us to standardize creating
+[Value Objects](https://martinfowler.com/bliki/ValueObject.html). Both those usages require
+something more than just a data structure. One of the commodities required for describing domain
+specifics is data validation. Spine builds on top of Protobuf with a Validation library.
+
+The purpose of the Validation library is:
+ - giving framework users an easy way to maintain simple domain invariants;
```

Comment:
Let's extract the common parts of the points.

I also think that mentioning Anticorruption layers and domain invariants is a bit too much. Both these statements require the reader to make several leaps from this guide through their experience towards realizing what the author meant in the first place.